plugins/shared/PluginManifest.cs

```diff
@@ -6,7 +6,7 @@ namespace Plugins.PluginShared;
 
 /// <summary>
 /// This class represents the OpenAI plugin manifest:
-/// https://platform.openai.com/docs/plugins/getting-started/plugin-manifest
+/// https://platform.openai.com/docs/plugins/getting-started
```

Comment:
Same as above.

Reply:
The old link now redirects to the new one.